frontend/src/api/ApiCaller.tsx

```diff
@@ -472,4 +472,26 @@ export class BackendAPICaller {
         })
         return result.content
     }
+
+    static async postSafePosition(installationCode: string) {
```

Comment:
Does this post a safe position since it calls the abort-current-missions-and-send-all-robots-to-safe-zone endpoint?

Reply:
No, will change the name